### PR TITLE
📖 Update docs for edge scheduler regarding kcp version

### PR DESCRIPTION
**Pull request by waltforme**
## Related issue(s)

Fixes #

### CODE REVIEW

File: docs/poc2023q1/edge-scheduler.md [hunk 1]
```diff
@@ -65,7 +61,7 @@ singleplacementslice.edge.kcp.io/test-1   5m54s
 ```
 
 Edit then delete the EdgePlacement.
-```
+```console
```
Comment (MikeSpreitzer) [inline docs/poc2023q1/edge-scheduler.md:64]: Is there a place where the meaning of "console" here is defined?

File: docs/poc2023q1/edge-scheduler.md [hunk 2]
```diff
@@ -1,12 +1,8 @@
 ### Edge scheduler
 The edge scheduler monitors the EdgePlacement, Location, and SyncTarget objects and maintains the results of matching.
 
-It starts from the kcp-scheduling-placement controller forked from kcp-dev/kcp, now being refactored towards the goal above.
-
-It works with [this snapshot](https://github.com/kcp-dev/kcp/tree/4506fdc064060b3fe82e1082533f9798b36ba7a5) of kcp.
-
 #### A short demo
-Start a kcp server with [this snapshot](https://github.com/kcp-dev/kcp/tree/4506fdc064060b3fe82e1082533f9798b36ba7a5).
+Start a kcp server, preferably with the version of `"github.com/kcp-dev/kcp"` listed in the `require` directive of go.mod.
```
Comment (MikeSpreitzer) [inline docs/poc2023q1/edge-scheduler.md:5]: I think this doc should say to use the version listed in `go.com`, without weakening it with words like "preferably"; we are taking no responsibility for making anything else work.

File: docs/poc2023q1/edge-scheduler.md (hunk 1 above)
Comment (waltforme) [reply docs/poc2023q1/edge-scheduler.md:64]: I think so.
https://github.com/github/linguist/blob/master/lib/linguist/languages.yml

File: docs/poc2023q1/edge-scheduler.md (hunk 2 above)
Comment (waltforme) [reply docs/poc2023q1/edge-scheduler.md]: Done. Thanks!

**Comment by clubanderson** [comment]
@MikeSpreitzer can you approve?

File: docs/poc2023q1/edge-scheduler.md (hunk 1 above)
Comment (MikeSpreitzer) [reply docs/poc2023q1/edge-scheduler.md:64]: Thanks!

**Review by MikeSpreitzer — APPROVED** [review]
/lgtm

**Pull request merged**